tools/deb_to_rb.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+set -e
+
+CREW_PREFIX=${CREW_PREFIX:-/usr/local}
+CREW_LIB_PREFIX=${CREW_PREFIX}/lib
+PKG_PREFIX=${CREW_LIB_PREFIX}/crew/packages
+GREEN='\e[1;32m';
+RED='\e[1;31m';
+BLUE='\e[1;34m';
+YELLOW='\e[1;33m';
+RESET='\e[0m'
+
+rm -rf ${CREW_PREFIX}/tmp/deb && cd ${CREW_PREFIX}/tmp/
+
+mkdir -p deb && cd ./deb
+
+echo -e "${YELLOW}Downloading deb...${RESET}"
+curl '-#' ${1} -o deb
+
+ar x deb
+
+
+tar xf control.tar.*
+  
+NAME=$(grep "Package: " ./control | sed -e 's/Package: //')
+PKGNAME=$(echo ${name} | sed -e 's/-/_/g')
+VER=$(grep "Version: " ./control | sed -e 's/Version: //' | sed -e 's/-/_/g')
+INFO=$(grep "Description: " ./control | sed -e 's/Description: //' | sed -e 's/-/_/g')
+SHA256SUM=$(sha256sum ./deb | cut -d ' ' -f1)
+
+OUT=${2}/${PKGNAME}.rb
+  
+cat <<EOF> ${OUT}
+require 'package'
+class ${PKGNAME^} < Package
+  description '${INFO}'
+  homepage ''
```

Comment:
Every package should include a homepage.  Maybe prompt for the url if it hasn't been passed as an argument?